.circleci/config.yml

```diff
@@ -583,6 +583,17 @@ commands:
             mvn test-compile --batch-mode -DskipTests -Dmaven.repo.local=<< parameters.m2_repo >>
 
 jobs:
+  build-api-docs-job:
```

Comment:
I have broken the apis docs several times now, so adding a check here so I don't do that again.